### ISSUE TITLE
Inconsistencies during say all with Word 365 UIA support

### ISSUE THREAD

**Issue by burakyuksek** [nvaccess/nvda#11988]
### Steps to reproduce:
1. Open a preferably large document.
2. Press NVDA+a to have NVDA read stuf
3. Either stop reading first or don't, then press some arrow key to change focus.
### Actual behavior:
Focus will go somewhere unrelated to the place say all stopped at.
### Expected behavior:
NVDA should be able to correctly track where it left in say all, and changing the currsor should reflect the position of where say all has last stopped.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2020.3
#### Windows version:
20h2
#### Name and version of other software in use when reproducing the issue:
Microsoft Office Word 365
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
All behaviour with UIA enabled was the ssame
#### If addons are disabled, is your problem still occuring?
I don't think it's relivent
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by bhavyashah** [comment]
I can confirm this issue with NVDA 2020.4 Beta 4 and Word 365. I initiated say all in a Word document that contains at least enough content to span multiple lines, pressed Ctrl to stop speech and mentally noted the last few words NVDA spoke, issued the read current line command, and observed that my focus was one line above where it should have been. In another test, my focus was two lines above where it should have been. To be clear, I was able to reproduce this bug in both browse and focus mode in Word. What sets my testing apart though is that Word UIA is disabled in NVDA's Advanced Settings.
